composer.json

```diff
@@ -1,11 +1,13 @@
 {
   "license": "GPL-3.0+",
   "require": {
+    "ext-intl": "*",
```

Comment:
That's for the number formatting I think. I did not know we could require PHP extensions through composer, so I ended up manually installing/enabling it my php.ini